readme.md

```diff
@@ -186,6 +186,7 @@ Made with Electron.
 - [Motrix](https://github.com/agalwood/Motrix) - Download manager.
 - [Franz](https://github.com/meetfranz/franz) - Skype, Slack, Hangouts, WhatsApp, Grape, Telegram, FB Messenger, Hipchat in the same app.
 - [Gmail Desktop](https://github.com/timche/gmail-desktop) - Unofficial Gmail app.
+- [Upcount](https://github.com/madisvain/upcount) - Invoice with ease. Open source, configurable & offline first.
```

Comment:
"Open source" is implied by it being in this list. I prefer a very short description of what the app is.

Reply:
@sindresorhus understood and made the description super simple "Easy Invocing app"